### PR TITLE
Exit bug

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### What does this implement/fix? Explain your changes.
- Exit bug fixed

### CODE REVIEW

File: nafas/functions.py [hunk 1]
```diff
@@ -221,6 +221,8 @@ def get_input_standard(input_func=input):
             try:
                 input_data[item] = int(input_func(""))
                 exit_flag = True
+            except KeyboardInterrupt:
+                sys.exit()
             except Exception:
                 print("[Error] Bad Input!")
```
Comment (sadrasabouri) [inline nafas/functions.py:227]: This is just to make an exception for `KeyboardInterrupt` when the user is inputting, right?
Isn't there any other way for that?
I think we should make it consistent with the other exception in the `main`:

1. Either `print("\n" + EXIT_MESSAGE)` or
2. `rais KeyboardInterrupt `

Inside the `except KeyboardInterrupt`

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Minor changes needed.

File: nafas/functions.py (hunk 1 above)
Comment (sepandhaghighi) [reply nafas/functions.py]: fixed

File: test/test.py [hunk 2]
```diff
@@ -70,6 +72,10 @@
 1- Beginner
 2- Medium
 3- Advanced
+>>> get_input_standard(test_keyboard_interrupt)
+Traceback (most recent call last):
+...
+SystemExit
```
Comment (sadrasabouri) [inline test/test.py:78]: Can we also have `EXIT_MESSAGE` content here?
That's not a big deal, just to make sure everything is tested as much as possible.

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
LGTM

File: test/test.py (hunk 2 above)
Comment (sadrasabouri) [reply test/test.py:78]: Rather than that, it's good. If you don't think adding such a thing is necessary you can merge it.

Comment (sepandhaghighi) [reply test/test.py:78]: Test system behavior is a bit weird here. I think it's because of the `sys.exit()`.

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**